README.md

```diff
@@ -1 +1,109 @@
-# leaderboard-with-webpack
\ No newline at end of file
+# leaderboard-with-webpack
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [leaderboard-with-webpack](#leaderboard-with-webpack)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[About the Project\] ](#-about-the-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Install](#install)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [📝 License ](#-license-)
+
+# 📖 [About the Project] <a name="about-project"></a>
+
+**[awesome books]** is a project that displays a list of books.
+
```

Comment:
- [ ] Kindly fix your project title and description.

  ![image](https://user-images.githubusercontent.com/81468675/217801123-90319960-b09d-42b2-85db-35cad11c9987.png)

Reply:
resolved! ✔️